docs/developer/extending/apps/local-apps-development.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Local apps development
```

Comment:
I think "app" should be singular here (and in the file name).

Reply:
changed